events/events.go

```diff
@@ -810,7 +810,7 @@ func NewGovernanceWorkloadCancelationMessage(id EventId, cause EndContractCause,
 
 	return &GovernanceWorkloadCancelationMessage{
 		GovernanceMaintenanceMessage: *govMaint,
-		Cause:                        cause,
+		Cause: cause,
```

Comment:
I think your formatter is off....you and Ling are having deuling formatters, or maybe Ling is off...